Redis module integration test fails in the 7.0 branch

Branch: 7.0 
link: https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/6210/beat=metricbeat,label=linux-immutable/console

```
11:29:00 ok  	github.com/elastic/beats/metricbeat/module/prometheus/collector	6.903s	coverage: 7.0% of statements
11:29:12 ok  	github.com/elastic/beats/metricbeat/module/redis/info	7.253s	coverage: 83.3% of statements
11:29:12 command [go test -race -tags integration -cover -coverprofile /tmp/gotestcover-878602352 github.com/elastic/beats/metricbeat/module/redis/key]: exit status 1
11:29:12 --- FAIL: TestFetch (5.10s)
11:29:12 	key_integration_test.go:72: SET NOAUTH Authentication required.
11:29:12 FAIL
11:29:12 coverage: 21.9% of statements
11:29:12 FAIL	github.com/elastic/beats/metri
```

--- a/metricbeat/module/redis/info/info_integration_test.go
+++ b/metricbeat/module/redis/info/info_integration_test.go
@@ -40,6 +40,9 @@ func TestFetch(t *testing.T) {
 	if err != nil {
 		t.Fatal("fetch", err)
 	}
+	if len(events) == 0 {
+		t.Fatal("no events")
+	}
 	event := events[0].MetricSetFields
 
 	t.Logf("%s/%s event: %+v", ms.Module().Name(), ms.Name(), event)

--- a/metricbeat/module/redis/metricset_integration_test.go
+++ b/metricbeat/module/redis/metricset_integration_test.go
@@ -37,6 +37,8 @@ const (
 )
 
 func TestPasswords(t *testing.T) {
+	t.Skip("Changing password affects other tests, see https://github.com/elastic/beats/issues/10955")
+
 	compose.EnsureUp(t, "redis")
 
 	registry := mb.NewRegister()

